hypothesis-python/tests/conjecture/common.py

```diff
@@ -428,3 +428,35 @@ def ir(*values: list[IRType]) -> list[IRNode]:
         )
 
     return tuple(nodes)
+
+
+def make_float_kw(
+    min_value,
+    max_value,
+    *,
+    allow_nan=True,
+    smallest_nonzero_magnitude=SMALLEST_SUBNORMAL,
+):
+    return {
+        "min_value": min_value,
+        "max_value": max_value,
+        "allow_nan": allow_nan,
+        "smallest_nonzero_magnitude": smallest_nonzero_magnitude,
+    }
+
+
+def make_integer_kw(min_value, max_value, *, weights=None, shrink_towards=0):
```

Comment:
Planned follow-up pull: use these convenience methods in tests! The space save of not specifying weights/shrink_towards/smallest_mag unless we care that it's non-default should be pretty solid.